src/commands/blocks.ts

```
@@ -403,7 +405,7 @@ export const blocksCommand = define({
 						const status = block.isActive ? pc.green('ACTIVE') : '';
 
 						const row = [
-							formatBlockTime(block, useCompactFormat),
+							formatBlockTime(block, useCompactFormat, ctx.values.locale),
```

ESLint

[error] Unsafe argument of type `any` assigned to a parameter of type `string | undefined`.

(ts/no-unsafe-argument)